Code/Write_output_file.py

```diff
@@ -1,19 +1,18 @@
+def write_to_file(output_filename, inst_hr, avg_hr, output):
+    """ Writes the output to a separate file
+
+    :param: output_filename: str, name for output
+    :param inst_hr: int instantaneous heart rate
+    :param avg_hr: int avg_heart_rate
+    :param output: string output
 
-def write_to_file(a, b, c):
-    import sys
-    """
-    Writes the output to a separate file
-    :param a: instantaneous heart rate
-    :param b: avg_heart_rate
-    :param c: output string
     """
+    import sys
     prev = sys.stdout
-    sys.stdout = open("output.txt", "w")
-    instantaneous_heart_rate = a
-    avg_heart_rate = b
-    threshold = c
-    print('Instantaneous Heart Rate is:', instantaneous_heart_rate)
-    print('Average Heart Rate is:', avg_heart_rate)
-    print('The condition for each window is', threshold)
+
+    sys.stdout = open(output_filename, "w")
```

Comment:
hmm what you're doing here is redirecting the stdout file descriptor to point to your file (as opposed to the default stdout that prints to your terminal) and are then restoring it. Generally this isn't the best way to write to files because it's sometimes unclear that you're writing to a file using `print`. Consult the python docs for canocial ways to write files using `f.write()`: https://docs.python.org/3/tutorial/inputoutput.html